examples/tests/nestedSuiteTest.spec.ts

```diff
@@ -50,7 +50,7 @@ test.describe('Top level suite', () => {
     ReportingApi.setDescription('Description for bottom level suite_1', 'Bottom level suite');
     ReportingApi.setDescription('Description for bottom level suite_2', 'Bottom level suite');
     ReportingApi.setTestCaseId('TestCaseIdForTheSuite', 'Bottom level suite');
-    test('Test should be passed', async ({ page }) => {
```

Comment:
Why are you decided to make it regular, not async function?

Reply:
I made all tests look similar. Do we need async here?

Reply:
The playwright provides e2e testing features, so it will be good to have some e2e tests samples in our agent example.